config/eloquent_driver.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Was there a specific reason why the config was changed from eloquent-driver to eloquent_driver?

Reply:
Jesse asked for it quite early on - to make it fit in with laravel conventions I believe

Reply:
Ah I see. It was this comment.

https://github.com/statamic/eloquent-driver/pull/42#discussion_r850778175

Sorry for the misunderstanding. He was asking for just the keys to be snake_case, not the file itself.

Would you please revert the filename back to `eloquent-driver.php`?
(As well as `config()` references, and the change to `MoveConfig.php`)

The convention is to have the filename be kebab-case but the keys be snake_case.

Examples:
- https://github.com/laravel/vapor-ui/blob/0ed2949c6a78a6fc85d9779a9c8c97462ff6ad33/src/VaporUiServiceProvider.php#L31
- https://github.com/spatie/laravel-query-builder/blob/10f7869790e749d7d25f9c8ec4801068e955ced2/config/query-builder.php#L27
- https://github.com/spatie/laravel-mail-preview/blob/59e3034b231f48abb6c34c1c1555fb3ce536ce9a/config/mail-preview.php#L13

(And yes I know we have a couple of snake_case config files in the core. live_preview.php and static_caching.php. Unfortunate oversight and a bit too late to change at this point.)

Reply:
Awesome, thanks.